src/averageAges.js

```diff
@@ -36,10 +43,18 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
-}
+  const women = people.filter(person => (
+    withChildren
+      ? people.find(child => child.mother === person.name)
+      : person.sex === 'f'
+  ));
+  const averageAge = women.map(age => age.died - age.born)
+    .reduce((sum, age) => sum + age);
 
+  return +(averageAge / women.length).toFixed(2);
```

Comment:
not sure what i like better - converting data types with `+...toFixed`

or longer, yet more math-style
`Math.round((averageAge / women.length) * 100) / 100`
```suggestion
  return +(averageAge / women.length).toFixed(2);
```